### PR TITLE
project type elligible

### PR DESCRIPTION
**Pull request by yunusfirat**

### CODE REVIEW

File: app/config/question-bank.js [hunk 1]
```diff
@@ -429,22 +429,65 @@ const questionBank = {
...
+              value: 'Replace an existing store that is no longer serviceable'
             },
             {
-              key: 'projectType'
-            }
+              key: 'project-type-A1',
+              value: 'Add a new store to increase existing capacity'
+            },
+            {
+              key: 'project-type-A1',
```
Comment (zavies97) [inline app/config/question-bank.js:480]: Keys should be unique here as well

**Review by zavies97 — CHANGES_REQUESTED** [review]
Looks good :) 

Making all the answer keys in project-type unique (project-type-A1, project-type-A2 etc) is the only thing I can see

Also, main has just been updated, so main should be merged into this branch

File: app/config/question-bank.js (hunk 1 above)
Comment (yunusfirat) [reply app/config/question-bank.js]: Thanks for the review 💯 
I have just pulled the main and pushed my new changes accordingly it's ready for your review. :)

**Review by zavies97 — APPROVED** [review]
Looks all good to me :)

**Pull request merged**